### PR TITLE
New iOS podspec

### PR DESCRIPTION
**Pull request by JamieDass**
There is a new podspec to use when integrating the PDFTron React Native Wrapper for iOS:
https://pdftron.com/downloads/ios/react-native/latest.podspec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,13 +156,18 @@ The release can be found here: https://github.com/PDFTron/pdftron-react-native/r
...
+
+**Please update your `Podfile` accordingly.**
 
 1. Open `Podfile` in the `ios` folder, add the following line to the `target 'MyApp' do ... end` block:
 
     ```
     target 'MyApp' do
         # ...
-        pod 'PDFNet', podspec: 'https://www.pdftron.com/downloads/ios/cocoapods/pdfnet/latest.podspec'
+        pod 'PDFNet', podspec: 'https://pdftron.s3.amazonaws.com/downloads/ios/react-native/latest.podspec'
```
Comment (dluco) [inline README.md:170]: Is the `s3.amazonaws` needed? Seems to work without it.
```suggestion
        pod 'PDFNet', podspec: 'https://pdftron.com/downloads/ios/react-native/latest.podspec'
```

File: example/ios/Podfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ platform :ios, '11.0'
 target 'example' do
   # PDFTron
   use_frameworks!
-  pod 'PDFNet', podspec: 'http://www.pdftron.com/downloads/ios/cocoapods/pdfnet/latest.podspec'
+  pod 'PDFNet', podspec: 'https://pdftron.s3.amazonaws.com/downloads/ios/react-native/latest.podspec'
```
Comment (dluco) [inline example/ios/Podfile:9]:
```suggestion
  pod 'PDFNet', podspec: 'https://pdftron.com/downloads/ios/react-native/latest.podspec'
```

File: scripts/newPodspecMessage.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import sys
+import os
+import shutil
+
```
Comment (dluco) [inline scripts/newPodspecMessage.py:4]: These aren't needed?
```suggestion
```

File: README.md (hunk 1 above)
Comment (JamieDass) [reply README.md:170]: Ah right, I just got it from Cyberduck. Should be ok to remove, will do it

File: scripts/newPodspecMessage.py (hunk 3 above)
Comment (JamieDass) [reply scripts/newPodspecMessage.py:4]: Oops right, lazily duplicated another script. Will remove.

**Review by dluco — APPROVED** [review]

**Pull request merged**